lib/Runtime/eachHelper.js

```diff
@@ -1,11 +1,23 @@
 import {CollectionComponent} from '../Components/CollectionComponent';
 
 export default function eachHelper(collection, options) {
-    let params = {
-        collection,
-        childTemplate: options.content,
-        emptyTemplate: options.inverse
-    };
+    // For simple array we don't need create wrapper
+    // if no additional params was not passed
+    if (Array.isArray(collection) && !Object.keys(options.hash).length) {
+        if (collection.length) {
```

Comment:
Well, in this case you need to duplicate in second condition. Maybe keep as it is